Fix dropbox issues

alistairmcmillan opened the pull request:
These changes get Dropbox working again in InkPad.

alistairmcmillan committed:
Add LSApplicationQueriesSchemes key for Dropbox

alistairmcmillan committed:
Fix Dropbox permission problem

RichardJones74 commented:
Is Dropbox working now?

alistairmcmillan commented:
Which version of InkPad do you have installed?

RichardJones74 commented:
The latest. I tapped import and got no response from the app.

michaeldugan commented:
Can you please update the files with the necessary fixes?

alistairmcmillan commented:
Sorry I'm not sure what you mean by the latest. Steve Sprang's Taptrix release of InkPad isn't available in the store anymore.

Can you point me to the one you have installed?

alistairmcmillan commented:
@michaeldugan Please publish the source for the version of Inkpad that you've released in the App Store (as you are required to do by the license) and I'll consider helping you out. :)

michaeldugan commented:
I have link to the files that I worked on. Just check my account.

RichardJones74 commented:
@alistairmcmillan Did you check on my issue? I put in my keys maybe thats a problem.

RichardJones74 commented:
Also I saw some version by a developer called "Envoza". It has an orange icon. It is up to date and has many features that weren't working in previous Inkpads that I tried. Does this person help you guys on this one or do they have their own source code files. I would like to help you guys work on this if you ever need help let me know!

alistairmcmillan commented:
@RichardJones74 I hadn't heard of Envoza before and I'm not sure if they have released the source, but if they have there is no evidence within this repo.

alistairmcmillan commented:
@michaeldugan I've checked your account. You have four public repos.
- https://github.com/michaeldugan/Inkpad is an old exact copy of Steve's repo with no changes
- https://github.com/michaeldugan/Ink-Pad-Pro1 is empty
- https://github.com/michaeldugan/Ink-Pad-Pro is empty
- https://github.com/michaeldugan/Inkpadpro contains five files
- A default .gitignore file
- A default README.md file
- A copy of NSArray+Additions.h with the copyright notice changed to substitute your own name for Steve Sprang (that's a no-no)
- A copy of NSArray+Additions.m with no changes
- A copy of WDAppDelegate.m with the copyright notice changed to substitute your own name for Steve
  Sprang (again, bad) and your Dropbox keys added

Is that all the changes you made to the app before releasing it on the store? Changed the copyright notices on the files, added your Dropbox keys and published?

Publish the entire code of Inkpad Pro as it is published on the App Store (as required by the license) and I'll tell you how to fix the Dropbox problem. :)

michaeldugan commented:
Does anyone know how to fix the Dropbox issues. I can't get it work when I open the app nothing happens when I press the button. Also I have open sourced my version on profile.

michaeldugan commented:
Dropbox still needs to fixed I am having some trouble with it

QuetzalPretzel commented:
@michaeldugan This might be an alien concept, but why don't you fix this yourself and contribute the fix back into this project?

alistairmcmillan commented:
@michaeldugan @QuetzalPretzel Weird thing about this... I download your code Michael and tried compiling it, and Dropbox works fine with no changes. Is the code you uploaded to GitHub the actual code of your app as it exists on the AppStore?

RichardJones74 commented:
Yes it's just recently that I can't get Dropbox to work. 

Sent from my iPad

> On Feb 28, 2016, at 9:51 AM, Alistair McMillan notifications@github.com wrote:
> 
> @michaeldugan @QuetzalPretzel Weird thing about this... I download your code Michael and tried compiling it, and Dropbox works fine with no changes. Is the code you uploaded to GitHub the actual code of your app as it exists on the AppStore?
> 
> —
> Reply to this email directly or view it on GitHub.

alistairmcmillan commented:
@RichardJones74 Do have an InkPad app in the store? If not, whose InkPad are you trying to use?

RichardJones74 commented:
@alistairmcmillan No nor do I have intentions to have one on the App Store. I just wanna learn how how to get it to work.

michaeldugan commented:
@alistairmcmillan The one I have on Github is the most current App Store version. I tried to fix some other bugs and broke Dropbox in the process and can't fix it.

alistairmcmillan commented:
@michaeldugan Let's discuss the problems with your app here. https://github.com/michaeldugan/Inkpad-Pro/issues/1
